src/apirequest.ts

```diff
@@ -307,3 +308,14 @@ class ProgressStream extends stream.Transform {
     callback();
   }
 }
+
+interface Headers {
+  // tslint:disable-next-line no-any
+  [index: string]: any;
+}
+
+function populateAPIHeader(headers: Headers) {
+  const pkg = require('../../package.json');
+  const nodeVersion = process.version.replace(/^v/, '');
```

Comment:
Loaded question - what happens when this runs in a browser?  We've had places in the code before that broke when we tried to directly use `process.version`

Reply:
Let's perhaps skip populating the header in the browser for the time being, there's no Node.js version to report, which makes the `gdcl` value unusable in isolation.

We could consider populating  `gl-web` but there's no straight forward way to consistently get the browser version -- so seems like a bit of a rabbit hole.

Reply:
So I'm ok with this approach, but wondering if we're being defensive enough.  I've seen so much weird shit with non-node environments trying to shim things, should we explicitly check each car along the chain?  It's ok if the answer is "you're paranoid"